src/test-utils.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { PropsWithChildren } from 'react'
+import { Provider } from 'react-redux'
+
+import {
+  AnyAction,
+  combineReducers,
+  configureStore,
+  EnhancedStore,
+  Middleware,
+  Reducer,
+  PreloadedState,
+} from '@reduxjs/toolkit'
+import { render, RenderOptions } from '@testing-library/react'
+
+import { setupStore, RootState, AppStore } from './state/store'
+
+export interface ExtendedRenderOptions extends Omit<RenderOptions, 'queries'> {
+  preloadedState?: PreloadedState<RootState>
+  store?: AppStore
+}
+
+export const swallowErrors = (testFn: () => void) => {
+  const error = console.error
+  console.error = () => {}
+  testFn()
+  console.error = error
+}
+
+// Return an object with the store and all of RTL's query functions
+export function renderWithStore(
+  ui: React.ReactElement,
+  {
+    preloadedState = {},
+    store = setupStore(preloadedState),
+    ...renderOptions
+  }: ExtendedRenderOptions = {}
+) {
+  function Wrapper({ children }: PropsWithChildren<unknown>): JSX.Element {
+    return <Provider store={store}>{children}</Provider>
+  }
+  // TODO add listeners
+
+  return { store, ...render(ui, { wrapper: Wrapper, ...renderOptions }) }
+}
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export function setupApiStore<
```

Comment:
Blir ikke `apiSlice` allerede satt opp i `setupStore`? Litt usikker på hva formålet er siden den ikke er brukt noe sted. Kan du vise hvordan den skal brukes i en test?

Reply:
Forskjellen var å kunne sende ønsket slice som param, istedenfor store som by default bruker combineReducer. Jeg har fjernet den i siste push da jeg klarte meg uten for nå